platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/MigrationCommand.java

```diff
@@ -86,4 +145,50 @@ protected void outputMessage(MigrationMessage msg) {
   protected PrintStream getConsole() {
     return System.out;
   }
+
+  @VisibleForTesting
+  protected String getSubjectName() {
+    return SubjectUtils.getName(SecurityUtils.getSubject(), null, true);
+  }
+
+  protected void postSystemNotice(String cmd) {
+    final String subjectName = getSubjectName();
+    final SystemNotice notice =
+        new SystemNotice(
+            getClass().getName() + '.' + System.currentTimeMillis(),
```

Comment:
Worst that will happen is that only one alert would show. Now that being said, it will create problems any way since there is only one zip file and only one system to restore to. During threat analysis we identified the need to limit the number of simultaneous exports and imports to 1. We have a ticket for that.